www/hockeyapp.js

```diff
@@ -20,6 +20,9 @@ var hockeyapp = {
     feedback: function (success, failure) {
         exec(success, failure, "HockeyApp", "feedback", []);
     },
+    composeFeedback: function (success, failure, attachScreenshot, data) {
+        exec(success, failure, "HockeyApp", "composeFeedback", [attachScreenshot === true || attachScreenshot === "true", JSON.stringify(data)]);
```

Comment:
Would it be better to check if the data parameter is specified and only JSON.stringify it if so? It just seems like a potentially clearer contract to the native side to treat the null as the absence of additional data, as opposed to an empty string.